src/tooltip.js

```diff
@@ -86,6 +86,10 @@ angular.module('cp-tooltip')
 					tooltipDisplayed = false;
 					if(tooltipEl) tooltipEl.remove();
 				}
+
+				function encodeHTML(str) {
+					return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
+				}
```

Comment:
@kentmclean good question, but I couldn't think of one that matched what I needed. I thought they were primarily for URI encoding - though they might work in this case too?